packages/create-svelte/shared/+prettier/.prettierrc

```
@@ -3,6 +3,7 @@
 	"singleQuote": true,
 	"trailingComma": "none",
 	"printWidth": 100,
+	"plugins": ["prettier-plugin-svelte"],
 	"pluginSearchDirs": ["."],
```

Maybe we should do 
```suggestion
	"plugins": ["./node_modules/prettier-plugin-svelte"],
```
instead? I'm using this hack for all my projects which works well when running `prettier` through the CLI or VSCode extension. The `--plugin-search-dir .` can also be removed too.

I made a comment at https://github.com/sveltejs/prettier-plugin-svelte/issues/155#issuecomment-812876576

---

would that work with yarn? also the search dir setting is likely needed if the users add more prettier plugins.

---

better safe than sorry - let's keep it as is

---

Hmm yeah my suggestion doesn't seem to work with yarn pnp (pnp also messes up vscode prettier plugin with this PR change and before). I guess we can go with this PR change still to have it partially working for yarn pnp.

> also the search dir setting is likely needed if the users add more prettier plugins.

Since they'd have to still add them in `plugins` with this PR, e.g. `"plugins": ["prettier-plugin-svelte", "prettier-plugin-something"]`, I don't think the `plugin-search-dir` would make a difference, besides not needing the `./node_modules/` part of my suggestion.

---

i prefer not having `node_modules` listed explicitly. I'd love to get rid of the `--plugin-search-dir .` on the cli though so it's a tough one. Ultimately this PR uses multiple ways to try and ensure the plugin is found. The only other option i know would be to use `prettierrc.cjs` and `require('prettier-plugin-svelte')`  to take resolving out of prettiers own hands. But thats also ugly.

I think the way this PR uses both json rc and cli option to ensure the plugin is found during api and cli use is working and has only one duplication, which is unfortunate but not that bad.

---

Let's go with this solution then